### ISSUE TITLE
Android: Selected State does not annonce when selected

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30842]
## Description
Does not announce "selected" on focus of:

- [x] Button #30954
- [x] TextInput #30955
- [ ] Slider #30956
- [x] Picker #30957
- [ ] ProgressBarAndroid #30958

## React Native version:
v0.63

## Expected Behavior
Upon focus of an element with the "selected" accessibilityState set to true, the focus announcement should say "selected". Upon change from selected to not selected, no additional announcement should be made, however on state change from unselected to selected, an announcement of "selected" should be made.  This works properly on some element, such as `<Text>`, and not on others, such as `<Button>`.

## Snack
https://snack.expo.io/NCALByZTa

## Android Details
The "selected" accessibilityState should correlate with Android's View's "selected" property, which seems to be properly working in most instances.  It is likely the components where this doesn't work have problems on the Javascript side, and not the Android side.

**Comment by sladyn98** [comment]
Looks like a good first issue. Any tips on how to get started with this like the files I should be looking at . Thanks

**Comment by amarlette** [comment]
I would recommend first looking at https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md if you haven't contributed to RN before. When it comes to where to start it looks like @intergalacticspacehighway has submitted a pull request for #30954 (pull request is  #31116) so their fixes for Button would be a good place to start since the other components have similar issues.

**Comment by huzaifaaak** [comment]
> This works properly on some element, such as , and not on others, such as .
@blavalla can you please update the components here ?

**Comment by blavalla** [comment]
> > This works properly on some element, such as , and not on others, such as .
> > @blavalla can you please update the components here ?

Sorry for the bad markdown! It was trying to interpret the React component names as HTML and just seemed to give up :) 

I updated the description to clarify.

**Comment by ishqelliot** [comment]
@amarlette all the listed issues are closed, is there anything remaining in this issue that needs to be picked?

**Comment by davwheat** [comment]
Bump - looks like all tasks are completed @amarlette
